kmod/core/Makefile

```diff
@@ -1,24 +1,29 @@
 # make rules
 KPATCH_BUILD ?= /lib/modules/$(shell uname -r)/build
 KERNELRELEASE := $(lastword $(subst /, , $(dir $(patsubst %/,%,$(KPATCH_BUILD)))))
+VERCODE := $(shell awk '/LINUX_VERSION_CODE/{print $$NF}' $(KPATCH_BUILD)/include/generated/uapi/linux/version.h)
 THISDIR := $(abspath $(dir $(lastword $(MAKEFILE_LIST))))
 
 ifeq ($(wildcard $(KPATCH_BUILD)),)
 $(error $(KPATCH_BUILD) doesn\'t exist.  Try installing the kernel-devel-$(KERNELRELEASE) RPM or linux-headers-$(KERNELRELEASE) DEB.)
 endif
 
+ifeq ($(shell if [ $(VERCODE) -lt 328192 ];then echo "y";fi),y)
 KPATCH_MAKE = $(MAKE) -C $(KPATCH_BUILD) M=$(THISDIR)
 
 kpatch.ko: core.c
 	$(KPATCH_MAKE) kpatch.ko
 
 all: kpatch.ko
+else
+all:
+$(info !!! Starting at v5.2, needed stacktrace routines are no longer available; not building kmod)
```

Comment:
When running make from the top-level dir, I see this message twice because it does a clean first.